codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/s3/S3ErrorIntegration.kt

```diff
@@ -92,28 +96,6 @@ class S3ErrorIntegration : SwiftIntegration {
             writer.dedent()
             writer.write("}")
         }
-
-        writer.openBlock("extension $operationErrorName {", "}") {
```

Comment:
Weird that this was all code generated.  Nothing in it is customized except the name of the type being extended.  Sounds basically like a protocol + extension... which you made it into.

Reply:
yeah i thought so too, there was much duplication of the same exact logic previously.